app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,30 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin" tools:context=".MainActivity">
+
+    <!--<TextView android:text="@string/hello_world" android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content" />-->
+
+    <EditText
+        android:id="@+id/inputtext"
+        android:layout_width="225dp"
+        android:layout_height="50dp"
+        android:layout_alignParentBottom="true"/>
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/list_button_text"
+        android:id="@+id/list_button"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentBottom="true"/>
+    <ListView
+        android:id="@+id/grocerylist"
+        android:layout_width="match_parent"
+        android:layout_height="400dp"
```

Comment:
Just like with strings, dimensions should go in dimens.xml. Also, on certain phones, constraining the height of the list view this way might mess up the layout a bit, but we'll cover stuff like that later in the course.